pkg/features/kube_features.go

```diff
@@ -210,12 +210,6 @@ const (
 	// Enable resource limits priority function
 	ResourceLimitsPriorityFunction utilfeature.Feature = "ResourceLimitsPriorityFunction"
 
-	// owner: @m1093782566
-	// beta: v1.9
-	//
-	// Implement IPVS-based in-cluster service load balancing
-	SupportIPVSProxyMode utilfeature.Feature = "SupportIPVSProxyMode"
```

Comment:
We can't remove this until after it is deprecated, so this needs a release-note "action required".  See "Rule #5b: CLI elements of admin-facing components" in https://kubernetes.io/docs/reference/using-api/deprecation-policy/

Reply:
Thanks for your response, Tim.

IIUC, we should keep the feature gateway for IPVS and just flip the `beta` to `GA` and then deprecate this after 1+ release cycles?